src/styles/blocks/calendar.scss

```
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+}
+
+/* .calendar styles */
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $days-gap;
+  width: ($day-width * 7) + ($days-gap * 7);
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  $week-days-list:
+    mon 1,
+    tue 2,
+    wed 3,
+    thu 4,
+    fri 5,
+    sat 6,
+    sun 7;
+
+  @each $week-day, $counter in $week-days-list {
+    &--start-day_#{$week-day} :first-child {
+      margin-left: ($day-width + $days-gap) * ($counter - 1);
+    }
+  }
+
+  @for $amount-of-days from 28 through 31 {
+    &--month-length_#{$amount-of-days} 
+    :nth-child(n + #{$amount-of-days + 1}) {
+      display: none;
+    }
+  }
+
+  /* day settings */
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+
+    width: $day-width;
+    height: $day-width;
+
+    cursor: pointer;
+    box-sizing: border-box;
+    border: 1px solid black;
+    background-color: #eee;
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+      }
+    }
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      transition: 0.5s;
```

```suggestion
      transition: transform 0.5s ease;
```
указывай свойство, которое анимируешь и timing-function
перенеси это правило в дефолтный класс  __day, потому что сейчас анимация работает только в одну сторону